lib/pymedphys/docs/libusers/ref/gui/index.rst

```diff
@@ -1,3 +1,4 @@
+========================
```

Comment:
I wonder if we could lint this sort of thing like we do with `black`...

Reply:
That was just a personal preference thing (trying to be consistent with heading syntax in the raw rst). Spinx automatically treats the highest level "syntax" as h1 and works out h2, h3, etc. from there anyway, so it's no biggy once rendered...